console/commands/composer.sh

```diff
@@ -38,6 +38,15 @@ if [[ "$#" != 0 && "$1" == "create-project" ]]; then
         exit 1
 fi
 
+COMPOSER_WORKDIR_PARAM=" --working-dir=${COMPOSER_DIR}"
+if [[ "$#" != 0 \
+    && ( $@ == *" -d "*  || $@ == *" -d="* \
+        || $@ == "-d "* || $@ == "-d="*  \
+        || $@ == *" --working-dir "* || $@ == *" --working-dir="*   \
+        || $@ == "--working-dir "* || $@ == "--working-dir="* ) ]]; then
+    COMPOSER_WORKDIR_PARAM=""
```

Comment:
I think here is better to just display an error message because it would be difficult to make it compatible with the execution of `composer validate` on line 55.

```suggestion
     printf "${COLOR_RED}Composer directory option not compatible with dockergento. This option is automatically set: ${COLOR_RESET}\n"
     echo ""
     echo "    --working-dir=${COMPOSER_DIR}"
     echo ""
     exit 1
```